buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -30,6 +32,71 @@
 
 FAIL_IF_MIGRATION_REQUIRED = os.environ.get("USE_BAZELISK_MIGRATE", "").upper() == "FAIL"
 
+REPO_PATTERN = re.compile(r"https?://github.com/(?P<owner>[^/]+)/(?P<repo>[^/]+).git")
+
+ISSUE_TEMPLATE = """Incompatible flag {flag} will break {project} once Bazel {version} is released.
+
```

Comment:
Nit: Maybe adding a comment says:
Please don't modify the issue title, it will prevent duplicate issue being filed.

Reply:
Done